server/plugin/api.go

```diff
@@ -1035,6 +1044,161 @@ func (p *Plugin) updateSettings(c *UserContext, w http.ResponseWriter, r *http.R
 	p.writeJSON(w, info.Settings)
 }
 
+func (p *Plugin) openAttachCommentIssueModal(c *UserContext, w http.ResponseWriter, r *http.Request) {
+	owner := r.FormValue("owner")
+	repo := r.FormValue("repo")
+	number := r.FormValue("number")
+	postID := r.FormValue("postId")
+	numberInt, err := strconv.Atoi(number)
+	if err != nil {
+		p.writeAPIError(w, &APIErrorResponse{Message: "Invalid param 'number'.", StatusCode: http.StatusBadRequest})
+		return
+	}
+	userID := r.Header.Get(HeaderMattermostUserID)
+	post, appErr := p.API.GetPost(postID)
+	if appErr != nil {
+		p.writeAPIError(w, &APIErrorResponse{ID: "", Message: "failed to load post " + postID, StatusCode: http.StatusInternalServerError})
```

Comment:
Check this everywhere
```suggestion
		p.writeAPIError(w, &APIErrorResponse{ID: "", Message: "failed to load the post " + postID, StatusCode: http.StatusInternalServerError})
```